src/main/java/de/muenchen/allg/itd51/wollmux/sidebar/layout/Layout.java

```diff
@@ -49,10 +57,53 @@ default void addControl(XControl control, int weight)
     addLayout(new ControlLayout(control), weight);
   }
 
+  Map<Layout, Integer> getLayouts();
+
+  XWindow getControl();
+
   /**
    * Die Höhe des Layouts.
    *
    * @return Die Höhe.
    */
   int getHeight();
+
+  public default XControl getControl(String id, Map<Layout, Integer> layouts)
```

Comment:
Ohne implementierung.
Der Code kommt in VerticalLayout und HorizontalLayout
Dann braucht man auch den Parameter layouts nicht.

Layout, ControlLayout, VerticalLayout und HorizontalLayout sind ein [Kompositum](https://de.wikipedia.org/wiki/Kompositum_(Entwurfsmuster))